tests/integration/test_sampling.py

```diff
@@ -136,3 +137,44 @@ def test_extended_sampling_resource(writer, tracer):
     tracer.configure(sampler=sampler, writer=writer)
     tracer.trace("should_not_send", resource=RESOURCE).finish()
     tracer.trace("should_send", resource="something else").finish()
+
+
+@snapshot_parametrized_with_writers
+def test_extended_sampling_tags(writer, tracer):
+    sampler = DatadogSampler(rules=[SamplingRule(0, tags=TAGS)])
+    tracer.configure(sampler=sampler, writer=writer)
+    tracer._tags = TAGS
+    tracer.trace("should_not_send").finish()
+    tracer._tags = {"banana": "True"}
+    tracer.trace("should_send").finish()
+
+
+@snapshot_parametrized_with_writers
```

Comment:
should we test `?` as well while we're at it?

Reply:
sure!